### DISCUSSION TITLE
突然変異の治療について

### DISCUSSION THREAD

**Discussion by ClocQ** [hengband/hengband#1178] [Ideas]
5/31朝のdiscordに話題として出たので念の為discussion立ち上げます。

突然変異の治療のためにズルまで行って戻るのは面倒、という意見が出ました。
解決策としては
(1) 他の街と同じく街移動できるようにする
(2) ズル以外の街でも突然変異を治せるようにする
という案が上がりました。

**Comment by dis-** [comment]
意図的に突然変異を治療するコストを上げているものと認識しています。
利便性を上げる場合、相応のコスト増加を与える必要があるでしょう。(例: テルモラでも治療できるが治療費が10倍等)

**Reply by ClocQ** [reply]
回答ありがとうございます。
話題になっていたので念の為discussionには挙げたものの、個人的にも修正が必要な程面倒かと言われると疑問に思っていたので納得の回答です。

開発側の見解提示に感謝します。
